pkl-codegen-java/src/main/kotlin/org/pkl/codegen/java/CliJavaCodeGeneratorOptions.kt

```diff
@@ -65,8 +65,12 @@ data class CliJavaCodeGeneratorOptions(
    */
   val renames: Map<String, String> = emptyMap()
 ) {
-  fun toJavaCodegenOptions() =
-    JavaCodegenOptions(
+  @Suppress("DeprecatedCallableAddReplaceWith")
+  @Deprecated("deprecated without replacement")
+  fun toJavaCodegenOptions() = toJavaCodeGeneratorOptions()
+
+  internal fun toJavaCodeGeneratorOptions() =
+    JavaCodeGeneratorOptions(
```

Comment:
Note: we should apply the same deprecation to `org.pkl.codegen.kotlin.CliKotlinCodeGeneratorOptions#toKotlinCodegenOptions`